### ISSUE TITLE
Aula 001 - Usando Git pela primeira vez

### ISSUE THREAD

**Issue by KlausSantos** [CollabCodeTech/forum-do-front-ao-end#18]
### Sair do Git Log sem fechar o Terminal
- No minuto 5:50 o Marco tentou fechou o log do git e não deu. Aqui eu usei apenas o comando:
```
: q
```
- Fica aí a dica.  
**OBS: Não sei qual label colocar kkk, e nem sei se ta certo abrir uma Issue para isto**

**Comment by Pliavi** [comment]
Exatamente, na verdade nem o dois pontos é necessário, apenas a tecla Q.

Aproveitando o caso, algumas vezes pode acontecer de dar o `git commit` e esquecer de digitar a mensagem ele poderá abrir:
- **Vim:** pra sair é necessário apertar a tecla ESC seguido de `:q!`
- **Nano:** pra sair deve estar escrito lá embaixo `^X`, significa CTRL+X
